oauth2/src/main/java/com/okta/spring/boot/oauth/env/OktaOAuth2PropertiesMappingEnvironmentPostProcessor.java

```diff
@@ -158,18 +161,14 @@ private PropertySource oktaRedirectUriPropertySource(Environment environment) {
     }
 
     private PropertySource oktaStaticDiscoveryPropertySource(Environment environment) {
-        String oauth2Suffix = Optional.ofNullable(environment.getProperty("okta.oauth2.issuer"))
-            .filter(issuer -> issuer.contains("/oauth2")) // Check if URL is a Custom Authorization Server
-            .map(s -> "") // if so, leave URL as is
-            .orElse("/oauth2"); // for the Okta Org Authorization Server, /oauth2 needs to be appended
 
         Map<String, Object> properties = new HashMap<>();
         properties.put("spring.security.oauth2.resourceserver.jwt.issuer-uri", "${okta.oauth2.issuer}");
-        properties.put("spring.security.oauth2.resourceserver.jwt.jwk-set-uri", "${okta.oauth2.issuer}" + oauth2Suffix + "/v1/keys");
-        properties.put("spring.security.oauth2.client.provider.okta.authorization-uri", "${okta.oauth2.issuer}" + oauth2Suffix + "/v1/authorize");
-        properties.put("spring.security.oauth2.client.provider.okta.token-uri", "${okta.oauth2.issuer}" + oauth2Suffix + "/v1/token");
-        properties.put("spring.security.oauth2.client.provider.okta.user-info-uri", "${okta.oauth2.issuer}" + oauth2Suffix + "/v1/userinfo");
-        properties.put("spring.security.oauth2.client.provider.okta.jwk-set-uri", "${okta.oauth2.issuer}" + oauth2Suffix + "/v1/keys");
+        properties.put("spring.security.oauth2.resourceserver.jwt.jwk-set-uri", "${" + OKTA_OAUTH_ISSUER_WITH_PATH + "}/v1/keys");
+        properties.put("spring.security.oauth2.client.provider.okta.authorization-uri", "${" + OKTA_OAUTH_ISSUER_WITH_PATH + "}/v1/authorize");
+        properties.put("spring.security.oauth2.client.provider.okta.token-uri", "${" + OKTA_OAUTH_ISSUER_WITH_PATH + "}/v1/token");
+        properties.put("spring.security.oauth2.client.provider.okta.user-info-uri", "${" + OKTA_OAUTH_ISSUER_WITH_PATH + "}/v1/userinfo");
+        properties.put("spring.security.oauth2.client.provider.okta.jwk-set-uri", "${" + OKTA_OAUTH_ISSUER_WITH_PATH + "}/v1/keys");
         properties.put("spring.security.oauth2.client.provider.okta.issuer-uri", "${okta.oauth2.issuer}"); // required for OIDC logout
```

Comment:
This should probably be replaced with the default discovery logic in the near future.  We just need to make sure (or force) the non OIDC usage (offline claim) to use discovery.